### PR TITLE
Disable unstable yum mirrors.iuscommunity.org repo

### PR DESCRIPTION
**Pull request by kokosing**
Disable unstable yum mirrors.iuscommunity.org repo

### CODE REVIEW

**Comment by kokosing** [comment]
This should fix automation

File: prestodev/centos6-presto-admin-tests-runtime/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ RUN \
     set -xeu && \
     # disable the notoriuosly unstable EPEL repo...
     # rm -rf /etc/yum.repos.d/epel* && \
+    # disable the unstable mirrors.iuscommunity.org repo...
```
Comment (sopel39) [inline prestodev/centos6-presto-admin-tests-runtime/Dockerfile]: uncomment or remove `rm -rf /etc/yum.repos.d/epel* && \`?

Comment (findepi) [reply prestodev/centos6-presto-admin-tests-runtime/Dockerfile]: Maybe the right approach would be to use something more official than `jdeathe/centos-ssh` as the base?
where does `iuscommunity.org` come from? (forgive if this is a silly q)

Comment (kokosing) [reply prestodev/centos6-presto-admin-tests-runtime/Dockerfile]: I agree, but please forgive my that I don't want to invest my time into presto admin infra right now.

**Pull request merged**